examples/command-help/commands-help.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+
+	argparse "../.."
```

Comment:
Local imports are not allowed in Go (while they work, it is not allowed by standards). It should import package as `github.com/...`